sr_utilities_common/scripts/sr_utilities_common/copyright_exclusions.cfg

```diff
@@ -1 +1 @@
-exclude_files=rewrite_bag_with_header_timestamps.py
+exclude_files=rewrite_bag_with_header_timestamps.py,timed_roslaunch.sh
```

Comment:
Change to:
`exclude_files=rewrite_bag_with_header_timestamps.py|timed_roslaunch.sh`